explorer/fuzzing/BUILD

```diff
@@ -143,7 +143,10 @@ cc_fuzz_test(
     testonly = 1,
     srcs = ["explorer_fuzzer.cpp"],
     corpus = glob(["fuzzer_corpus/*"]),
-    shard_count = 8,
+    # TODO: This takes several minutes to test the corpus. Either we need to
+    # trim the corpus down to reduce execution time, or find a way to shard
+    # execution.
+    tags = ["manual"],
```

Comment:
Worth filing a feature request and linking it here?

Reply:
Noting that you're objecting to the switch, I'd like to see if we can make progress here before working on a fix. That's why there's a TODO.

Reply:
Sure and this isn't blocking. I'm not suggesting working on a fix ourselves necessarily, upstream may be responsive to a feature request.